.changeset/silent-buckets-stop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": minor
+---
+
+Added new rule `noUnassignedVariables`, which disallow let or var variables that are read but never assigned.
```

Comment:
Please make a link of the rule name.

```suggestion
Added new rule [`noUnassignedVariables`](https://biomejs.dev/linter/rules/no-unassigned-variables/), which disallows `let` or `var` variables that are read but never assigned.
```

Also add a few examples of valid and invalid code, please.